hive/envs/gym_env.py

```diff
@@ -54,19 +54,20 @@ def create_env_spec(self, env_name, **kwargs):
         )
 
     def reset(self):
-        observation = self._env.reset()
+        observation, _ = self._env.reset(seed=self._seed)
+        self._seed = None
         return observation, self._turn
 
     def step(self, action):
-        observation, reward, done, info = self._env.step(action)
+        observation, reward, terminated, truncated, info = self._env.step(action)
         self._turn = (self._turn + 1) % self._num_players
-        return observation, reward, done, self._turn, info
+        return observation, reward, terminated, truncated, self._turn, info
 
     def render(self, mode="rgb_array"):
```

Comment:
[New render method in gym](https://github.com/openai/gym/blob/6a04d49722724677610e36c1f92908e72f51da0c/gym/core.py#L153) does not have the`mode` parameter. Update this accordingly, and make sure it works.

Reply:
Yeah the mode needs to be removed and we decided that it will be done in a separate pr.

Reply:
Sorry for the confusion; I may not have stated things correctly. This line is a bug and will influence others' works. Hence, it makes the code incomplete for me, and I am not going to accept it. And, also I don't think postponing bugs to the next PRs is a good strategy in general. (Maybe you want to correct me @apsarath @dapatil211)

Based on your comment on my change request, I think you knew the bug existed before, but unfortunately, I cannot find a single comment/description discussing this issue. This behavior is very problematic because the general assumption for the reviewers is that the code is bugless, and the code's author discusses any notable things.

By the way, who are you referring to when you say *We decided ...*?